HLTrigger/JetMET/plugins/HLTHFJetCleaner.cc

```diff
@@ -0,0 +1,268 @@
+// -*- C++ -*-
+//
+// Package:    HLTrigger/HLTHFJetCleaner
+// Class:      HLTHFJetCleaner
+//
+/**\class HLTHFJetCleaner HLTHFJetCleaner.cc HLTrigger/HLTHFJetCleaner/plugins/HLTHFJetCleaner.cc
+
+ Description: [one line class summary]
+
+ Implementation:
+     [Notes on implementation]
+*/
+//
+// Original Author:  Alp Akpinar
+//         Created:  Wed, 01 Dec 2021 15:47:19 GMT
+//
+//
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "DataFormats/JetReco/interface/PFJet.h"
+#include "DataFormats/METReco/interface/CaloMET.h"
+#include "DataFormats/Math/interface/deltaPhi.h"
+
+//
+// class declaration
+//
+
+class HLTHFJetCleaner : public edm::stream::EDProducer<> {
+public:
+  typedef std::vector<reco::PFJet> JetCollection;
+  explicit HLTHFJetCleaner(const edm::ParameterSet&);
+  ~HLTHFJetCleaner() override;
```

Comment:
```suggestion
  ~HLTHFJetCleaner() override = default;
```

Reply:
Implemented in [1cff316](https://github.com/cms-sw/cmssw/pull/37539/commits/1cff316fd36bd151dc2ad6f329f7dfd8e9351b2d)